packages/eslint-config-airbnb-base/index.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  extends: [
+    './legacy',
+    './rules/es6',
+  ].map(require.resolve),
+  parserOptions: {
+    ecmaVersion: 7,
+    sourceType: 'module',
+  },
+  rules: {
+    strict: 2,
```

Comment:
Specifically, this isn't "base", it's "Airbnb base". It matches what we use, minus React.